app/_components/commodities/components/GridBodyItemMobile.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowUp, faArrowDown } from '@fortawesome/free-solid-svg-icons';
+import { formatThousands } from '@/app/_hooks/textFormatting';
+import { Center, Flex, Icon, Tbody, Td, Text, Tr } from '@chakra-ui/react';
+import { ICommodityFormResponse } from '@/app/_types/commodity';
+import { calculateTimeDifference, renderStationTypeIcon } from '../helpers';
+import GetColor from '@/app/_hooks/colorSelector';
+
+interface IGridBodyItemProps {
+  isDark: boolean;
+  commodity: ICommodityFormResponse;
+  isBuying: boolean;
+}
+
+const GridBodyItemMobile: React.FC<IGridBodyItemProps> = ({
+  isDark,
+  commodity,
+  isBuying,
+}) => {
+  const [showItemCard, setShowItemCard] = useState(false);
+
+  const toggleItemCard = () => (
+    <Tr fontSize="sm" borderBottomRadius="10px">
+      <Td colSpan={4} lineHeight={2}>
+        <Text>Station Name: {commodity.station.name}</Text>
+        <Flex gap={2}>
+          Station Type:
+          {renderStationTypeIcon(commodity.station, isDark)}
+        </Flex>
+        <Text>
+          Arrival Distance: {commodity.station.arrivalDistance ?? '?'} ls
+        </Text>
+        <Text>
+          Last Updated: {calculateTimeDifference(commodity.priceUpdatedAt)}
+        </Text>
+      </Td>
+    </Tr>
+  );
+
+  return (
+    <Tbody
+      cursor="pointer"
+      onClick={() => setShowItemCard(!showItemCard)}
+      _odd={{
+        background: `${GetColor('grid-accent')}`,
+      }}
+    >
+      <Tr>
```

Comment:
Similar to desktop, alignments are inconsistent on mobile.  Headers all align left, content cells all align right.

Reply:
Fixed:
@500px - a few updates: 1) alignment of distance heading and grid element changed. 2) dropped the word 'Sell ' or 'Buy ' from the 'Price' heading. 3) system name now has scrolling for long names (see pic 2), as a result I included the system name again in the dropdown card element

![Screenshot 2023-09-04 at 15-00-47 EDPN - Commodities](https://github.com/ed-pilots-network/frontend/assets/67390534/2bfbdebc-91ed-4d7f-ad64-d8fceac26aa0)

![Screenshot 2023-09-04 at 14-53-53 EDPN - Commodities](https://github.com/ed-pilots-network/frontend/assets/67390534/9fea8ae5-aa9f-41f4-b10d-fb9e4939ba5c)

Reply:
I feel like I'm coming across as nitpicky so I'm going to approve and you can either look into this now or we can just deal with it another time, but the alignment here is still making me a bit itchy. 

It looks like the content cells have some padding that the heading cells don't. So even though the alignment is technically fixed, price/supply/system content cells look about 10px out of alignment. The distance heading I can't tell if its aligned right, but if it is it's 2px out of alignment with the content.

![image](https://github.com/ed-pilots-network/frontend/assets/319883/f94024bd-3e1b-4e52-bb08-07b121df6087)